src/coreclr/jit/assertionprop.cpp

```diff
@@ -4358,6 +4358,7 @@ GenTree* Compiler::optAssertionPropLocal_RelOp(ASSERT_VALARG_TP assertions, GenT
 
     op2->AsIntCon()->gtIconVal = foldResult;
     op2->gtType                = TYP_INT;
+    op2->ClearIconHandleMask();
```

Comment:
nit: all 3 lines can be replaced with `op2->BashToConst(foldResult, TYP_INT);`

Reply:
Thanks, I was wondering if there's a pattern!